File: Retrospective_Core/Models/RetroFamily.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Retrospective_Core.Models
{
    public class RetroFamily
    {
	    public ICollection<RetroCard> RetroCards { get; set; } = new List<RetroCard>();

		[Key]
		public int Id { get; set; }
		public string Content { get; set; }
		public int Position { get; set;  }

		[ForeignKey("RetroColumnId")]
		public int RetroColumnId { get; set; }

		public virtual RetroColumn RetroColumn { get; set; }
	}
}

File: Retrospective_Front_End/src/models/RetroFamily.ts
import {BaseItem} from './BaseItem';
import {RetroCard} from './RetroCard';

export class RetroFamily extends BaseItem {

    constructor(
        public id: number,
        public content: string,
        public position: number,
        public retroColumnId: number,
        public retroCards: RetroCard[]
    ) {
        super(id, content, position, retroColumnId);
    }
}


